tests/unit/test_mixins.py

```diff
@@ -377,6 +373,85 @@ def test_is_admin_grader(self, preferences, expected_is_admin_grader):
 
 
 @ddt.ddt
+class TestAuthXBlockMixin(TestCase, TestWithPatchesMixin):
+
+    USER_ID = 1234
+    COURSE_ID = 4321
+
+    class AuthXBlockMixinGuineaPig(AuthXBlockMixin):
+
+        @property
+        def course_id(self):
+            return 4321
+
+        @property
+        def see_dashboard_for_all_orgs_perms(self):
+            return set(["all_orgs_perm"])
+
+        @property
+        def ta_roles(self):
+            return set(["ta_role"])
+
+        @property
+        def see_dashboard_ta_perms(self):
+            return set(["ta_perm"])
+
+        @property
+        def see_dashboard_role_perms(self):
+            return set(["single_role_perm"])
```

Comment:
They have no actual meaning outside of this tests, the problem is that there is no good context where I can store them, if I store them in `AuthXBlockMixinGuineaPig` they won't be visible on ``ddt.data``, if I store them `TestAuthXBlockMixin` they won't be visible in  `AuthXBlockMixinGuineaPig` (as well as ``ddt.data``) 

I'd say to leave them as it is, only solution I come up with is to move them to `TestAuthXBlockMixin`, and then inject `TestAuthXBlockMixin` instance to `AuthXBlockMixinGuineaPig`.